src/frontend/web_application/src/components/DropdownMenu/style.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/common';
+@import '../../styles/object/o-triangle';
+
+.m-dropdown-menu {
+  background-color: $co-color__primary--lower;
+
+  &::before {
```

Comment:
:100: agree but it's not required for now

And in the same way, triangle position should stick with `alignment` property (but not useful too)